Dockerfile

```diff
@@ -0,0 +1,26 @@
+# custom cinder-volume container - having python-3parclient
+FROM registry.redhat.io/rhosp-rhel8/openstack-cinder-volume:16.0-77
```

Comment:
@traghavendra  I think you can drop the tag number from the image.

You can refer https://github.com/hpe-storage/hpe-3par-cinder-rhosp13/blob/master/Dockerfile#L2 this for the same.

Reply:
@sneharai4 
Earlier we didn't use tag (16.0-77), but then image was not getting built.
We also tried with "latest" tag, but no luck.
So, finally used this tag "16.0-77".

Reply:
@traghavendra I am fine with the version. Since RH registry doesn't maintain latest tag.
You can check here https://catalog.redhat.com/software/containers/detail/5de6b086d70cc51644a56e00?tag=all